libs/base/eventcontext.ts

```diff
@@ -86,7 +86,7 @@ namespace control {
                 EventContext.lastStats = `fps:${Math.round(fps)}`;
                 if (fps < 99)
                     EventContext.lastStats += "." + (Math.round(fps * 10) % 10)
-                if (control.profilingEnabled()) {
+                if (control.ramSize() > 2000000 && control.profilingEnabled()) {
```

Comment:
should 2000000 be pulled from a bootloader constant?

Reply:
nah, if you have over 2M of memory, the dmesg is probably going to be big enough